frontend-v2/src/features/data/LoadDataStepper.tsx

```diff
@@ -239,6 +239,17 @@ const LoadDataStepper: FC<IStepper> = ({ csv = "", onCancel, onFinish }) => {
     }
   };
 
+  const onResetDataset = () => {
+    setData([]);
+    setErrors([]);
+    setWarnings([]);
+    setTimeUnit(undefined);
+    setAmountUnit(undefined);
+    setPrimaryCohort("Group");
+    setFileName("New Dataset");
+    restart();
+  };
```

Comment:
There's a use case where I might press cancel, on the very first step, because I accidentally clicked 'upload data' by mistake. In that case, I want to close the uploader and go back to viewing my current dataset, rather than resetting the upload stepper. I'm not sure if this handler will handle that case.

There's an `onCancel` callback which will handle that case, but I'm not sure how it will get called with the new controls.